### ISSUE TITLE
GitGutterNextHunk and GitGutterPrevHunk is slow with specific file types

### ISSUE THREAD

**Issue by Dosx001** [airblade/vim-gitgutter#805]
> What is the latest commit SHA in your installed vim-gitgutter?
256702dd1432894b3607d3de6cd660863b331818
> What vim/nvim version are you on?
Vim 8.1.2269

I was working on a python file when I noticed ```GitGutterNextHunk``` and ```GitGutterPrevHunk``` being slow. 

I tested few a few file types on different git repos and for the following file types, ```GitGutterNextHunk``` and ```GitGutterPrevHunk``` work perfectly.
.cpp .ts .js .sh .scss .css .json .md

For the following file types, ```GitGutterNextHunk``` and ```GitGutterPrevHunk``` are slow.
.vim .py

I got the same result even when opening two file types at the same time. 
e.g. ```vim file.py file.cpp -p```
The .cpp file is fine but the .py file is slow.

**Comment by airblade** [comment]
The only way to diagnose this by [profiling it](https://github.com/airblade/vim-gitgutter/wiki/How-to-profile-vim-gitgutter).  Please could you do that and upload the profile here?

**Comment by airblade** [comment]
The plugin doesn't do anything filetype-specific.  I can only imagine that either your git-diff is slower with Vimscript and Python files for some reason or that you have Vimscript and Python filetype plugins which are somehow slowing things down.

The only way to tell is by profiling.

**Comment by Dosx001** [comment]
@airblade I am so sorry for the late reply. I was been busy since I made this issue. 

I figured out the problem. It is being caused by the same reason as this issue #797.

To resolve both #797 and #805, I decided just settle with this.
```nmap <F10> <Plug>(GitGutterNextHunk)```
```nmap <F9> <Plug>(GitGutterPrevHunk)```

A little awkward but no more problems. I originally trying using <C-]> and <C-[> but it causing some very strange behavior.

Sorry for all these issues. I just don't like it when I am trying to move between hunks, I have to press two different keys back to back. I don't mind pressing two keys at the same time like ctrl + ], but seem like that doesn't work. Sad day.

Not sure if you get notified when someone comments on your repo. So I let you close this issue.
